Active trains

salva-maiorano opened the pull request:
- Oct-2018
improve active train location
move trains smoothly to the new position
station info when selected
more space for the map
change train icons
debug mode when clicking the legend station icon

Commit:
improve active train location

Commit:
no message

brendannee commented:
This is awesome - I updated the whereisbart.com site with your changes. 

I updated the station info so that it shows up in the map popup window when clicked - that makes it a bit more noticeable that in the sidebar.

Thanks for the pull request, feel free to post more if you find additional things you'd like to add.

salva-maiorano commented:
cool!
glad you like it. i enjoyed coding it.
nice move of the station info to the popup i looks much better this way
take care
/s


On Mon, Oct 15, 2018 at 11:14 PM Brendan Nee <notifications@github.com>
wrote:

> This is awesome - I updated the whereisbart.com site with your changes.
>
> I updated the station info so that it shows up in the map popup window
> when clicked - that makes it a bit more noticeable that in the sidebar.
>
> Thanks for the pull request, feel free to post more if you find additional
> things you'd like to add.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/brendannee/whereisbart.com/pull/4#issuecomment-430113920>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AWGgGtxIyqj4MGzdXdyk6TyThVdSLOsvks5ulXlUgaJpZM4XdfZp>
> .
>

salva-maiorano commented:
Hey Brendan
i made some other changes to code
take them a look and let me know what you think
tc
/s

On Tue, Oct 16, 2018 at 12:01 AM Salva Maiorano ***@***.***>
wrote:

> cool!
> glad you like it. i enjoyed coding it.
> nice move of the station info to the popup i looks much better this way
> take care
> /s
>
>
> On Mon, Oct 15, 2018 at 11:14 PM Brendan Nee ***@***.***>
> wrote:
>
>> This is awesome - I updated the whereisbart.com site with your changes.
>>
>> I updated the station info so that it shows up in the map popup window
>> when clicked - that makes it a bit more noticeable that in the sidebar.
>>
>> Thanks for the pull request, feel free to post more if you find
>> additional things you'd like to add.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/brendannee/whereisbart.com/pull/4#issuecomment-430113920>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AWGgGtxIyqj4MGzdXdyk6TyThVdSLOsvks5ulXlUgaJpZM4XdfZp>
>> .
>>
>
